### PR TITLE
fix(updater): `format!` panic on `c_long`

### PR DESCRIPTION
**Pull request by felangel**
## Description

- fix(updater): `format!` panic on `c_long`

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [X] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: library/src/c_api.rs [hunk 1]
```diff
@@ -33,15 +33,15 @@ pub struct AppParameters {
     pub cache_dir: *const libc::c_char,
 }
 
-fn to_rust(c_string: *const libc::c_char) -> String {
+fn string_to_rust_string(c_string: *const libc::c_char) -> String {
```
Comment (eseidel) [inline library/src/c_api.rs:36]: Why string_to_rust_string if the other is to_rust_vector? 🙃

File: library/src/c_api.rs [hunk 2]
```diff
@@ -68,9 +64,10 @@ fn app_config_from_c(c_params: *const AppParameters) -> updater::AppConfig {
 /// configuration compiled into the app.
 #[no_mangle]
 pub extern "C" fn shorebird_init(c_params: *const AppParameters, c_yaml: *const libc::c_char) {
+    print!("in shorebird_init");
```
Comment (eseidel) [inline library/src/c_api.rs:67]: You won't want this for several reasons.
1.  you would want println not print.  print won't add a \n iirc.
2. print won't show up on android, only logs will 😭 
3. this is probably just leftover debugging. :)

Comment (felangel) [reply library/src/c_api.rs:67]: yeah I was just trying to debug/sanity check wasn't planning to check this in.

**Comment by eseidel** [comment]
I believe we can close this?

**Pull request closed without merging**